Jellyfin.Server/Migrations/MigrationOptions.cs

```diff
@@ -0,0 +1,23 @@
+namespace Jellyfin.Server.Migrations
+{
+    /// <summary>
+    /// Configuration part that holds all migrations that were applied.
+    /// </summary>
+    public class MigrationOptions
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MigrationOptions"/> class.
+        /// </summary>
+        public MigrationOptions()
+        {
+            Applied = System.Array.Empty<string>();
+        }
+
+#pragma warning disable CA1819 // Properties should not return arrays
+        /// <summary>
+        /// Gets or sets he list of applied migration routine names.
+        /// </summary>
+        public string[] Applied { get; set; }
```

Comment:
Why not use a list here?

Reply:
Because I don't know how well serialize-deserialize routines handle that... I can experiment.
What's wrong with it being an array here, though?

Reply:
Mostly just asking because I thought you might have some insight into how the configuration system works since I know nothing :P The benefits of a list would be removing the `#pragma warning disable` and no need to convert to/from `List` in the code that uses this property. Not going to hang up an approval on this though